src/Publish/Microsoft.NET.Sdk.Publish.Targets/netstandard1.0/PublishTargets/Microsoft.NET.Sdk.Publish.MSDeploy.targets

```diff
@@ -155,10 +155,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <PropertyGroup>
       <_EnableRuleList>$(MsDeployDoNotDeleteRule)</_EnableRuleList>
```

Comment:
yes, it makes sense for this to be item. In the old targets these were defined as props but now is a good time to fix these up :) . Will update the PR.